src/screens/block/block.vue

```diff
@@ -122,7 +86,7 @@ import RelativeTime from '../../components/relativeTime'
 import Transaction from '../../components/transaction/transaction'
 import Loader from '../../components/loader'
 
-const blockHashRegex = RegExp('^bh\\$[1-9A-HJ-NP-Za-km-z]{48,49}')
+const blockHashRegex = RegExp('^kh|mh')
```

Comment:
Does block hash format change significantly? Can you bring `$[1-9A-HJ-NP-Za-km-z]{48,49}` back with a proper length? Also, `$` at the end is necessary to much the end of input.